users/templates/base.html

```diff
@@ -6,56 +6,154 @@
+    <div class="href-blocks">
+        <a href={{ custom_projects_list_url }}>
+            {% if request.path == custom_projects_list_url %}
+            <div class="img"><img src="{% static 'users/images/white/iko-04.png' %}"></div>
+            <div class="text" style="color:white;">{% trans 'Projects' %}</div>
+            {% else %}
+            <div class="img"><img
+                    src="{% static 'users/images/grey/ico-04.png' %}"></div>
+            <div class="text">{% trans 'Projects' %}</div>
+            {% endif %}
+        </a>
+        <div style="clear:both;"></div>
+    </div>
+    {% endif %}
+    <div class="href-blocks">
+        <a href="{{ custom_report_list_url }}">
+            {% if request.path == custom_report_list_url %}
+            <div class="img"><img
+                    src="{% static 'users/images/white/iko-05.png' %}"></div>
+            <div class="text" style="color: white;">{% trans 'My reports' %}</div>
+            {% else %}
+            <div class="img"><img
+                    src="{% static 'users/images/grey/ico-05.png' %}"></div>
+            <div class="text">{% trans 'My reports' %}</div>
+            {% endif %}
+        </a>
+        <div style="clear:both;"></div>
+    </div>
+    <div class="href-blocks" id="logout">
+        <a href="{% url 'logout' %}">
+            <div class="img"><img
+                    src="{% static 'users/images/grey/ico-23.png' %}"></div>
+            <div class="text">
+                {% trans 'Logout' %}
+            </div>
+        </a>
+        <div style="clear:both;"></div>
+    </div>
+    <div id="logo-menu">
+        <span><input type="image" onclick="displayNav()" src="{% static 'users/images/logo.png' %}"/>
+        <img id="sheetstorm" src="{% static 'users/images/ico-32.png' %}"/></span>
+    </div>
+</div>
+{% endif %}
+<div id="page-content-wrapper">
     {% block content %}
     {% endblock %}
 </div>
-  </main>
+
+<script>
+
+
+function displayNav() {
+ text_div_styles = document.getElementsByClassName("text");
+ menu_styles = document.getElementById("menu").style;
+ user_type_div_styles = document.getElementById("user-type").style;
+ sheetstorm_div_styles = document.getElementById("sheetstorm").style;
+ avatar_div_styles = document.getElementById("avatar").style;
+if (menu_styles.width == "15%" || menu_styles.width == "") {
+    user_type_div_styles.visibility = "hidden";
+    menu_styles.width = "8%";
+    sheetstorm_div_styles.display = "none";
+    avatar_div_styles.marginLeft = "-2.1rem";
+    for (i = 0; i < text_div_styles.length; i++) {
+        text_div_styles[i].style.display = "none";
+    }
+} else {
+    menu_styles.width = "15%";
+    user_type_div_styles.visibility = "visible";
+    sheetstorm_div_styles.display = "inline-block";
+    avatar_div_styles.marginLeft = "0";
+    for (i = 0; i < text_div_styles.length; i++) {
+        text_div_styles[i].style.display = "block";
+    }
+  }
+}
+
+
+
+</script>
```

Comment:
Please move this script to `.js` file. HTML files should be have only html "code" :D

Reply:
Done